### PR TITLE
Adding ability to add to webpack node config

### PR DESCRIPTION
**Pull request by mishkinf**

### CODE REVIEW

**Comment by mishkinf** [comment]
@cybrass can you take a look at this change? I needed the ability to add to the node config section of the webpack config to make certain changes for BP.

File: templates/new/src/config/.Dockerfile [hunk 1]
```diff
@@ -7,4 +7,4 @@ FROM truecar/gluestick:0.10.8
 ADD . /app
 
 RUN yarn
-RUN ASSET_URL=__GS_ASSET_URL__ gluestick build
+RUN gluestick build
```
Comment (toddw) [inline templates/new/src/config/.Dockerfile]: We don't want this line change

**Comment by mishkinf** [comment]
This change enables a client app to ignore certain node modules which seem to cause issues. This is accomplished by something akin to... 
`module.exports = {
  additionalLoaders: [],
  additionalPreLoaders: [],
  additionalAliases: {},
  additionalWebpackNodeConfig: {
    module: "empty",
    fs: "empty",
    net: "empty"
  },
  plugins: [],
  vendor: []
};` which is caused by errors such as this.. 
`ERROR in ./~/babel-core/lib/helpers/resolve.js
Module not found: Error: Cannot resolve module 'module' in /Users/mfaustini/workspace/gluestick-addon-battery-pack/node_modules/babel-core/lib/helpers
 @ ./~/babel-core/lib/helpers/resolve.js 34:14-31

ERROR in ./~/debug/node.js
Module not found: Error: Cannot resolve module 'net' in /Users/mfaustini/workspace/gluestick-addon-battery-pack/node_modules/debug
 @ ./~/debug/node.js 174:16-30
`

File: templates/new/src/config/.Dockerfile (hunk 1 above)
Comment (mishkinf) [reply templates/new/src/config/.Dockerfile]: hmm interesting. wonder how this line changed. let me revert it

**Comment by toddw** [comment]
:+1:

**Pull request merged**